packages/admin-next/dashboard/src/routes/product-variants/product-variant-manage-inventory-items/components/manage-variant-inventory-items-form/manage-variant-inventory-items-form.tsx

```diff
@@ -0,0 +1,301 @@
+      (i) => (existingItems[i.inventory.id] = i.required_quantity)
+    )
+
+    values.inventory.forEach((i) => (selectedItems[i.inventory_item_id] = true))
+
+    const payload = {
+      create: [],
+      update: [],
+      delete: [],
+    }
+
+    values.inventory.forEach((v) => {
+      if (v.inventory_item_id in existingItems) {
+        if (v.required_quantity !== existingItems[v.inventory_item_id]) {
+          payload.update.push({
+            required_quantity: v.required_quantity,
+            inventory_item_id: v.inventory_item_id,
+            variant_id: variant.id,
+          })
+        }
+      } else {
+        payload.create.push({
+          required_quantity: v.required_quantity,
+          inventory_item_id: v.inventory_item_id,
+          variant_id: variant.id,
+        })
+      }
+    })
+
+    variant.inventory_items.forEach((i) => {
+      if (!(i.inventory.id in selectedItems)) {
+        payload.delete.push({
+          inventory_item_id: i.inventory.id,
+          variant_id: variant.id,
+        })
+      }
+    })
+
+    for (const k in payload) {
+      if (!payload[k].length) {
+        delete payload[k]
+      }
+    }
+
+    await mutateAsync(payload, {
+      onSuccess: () => {
+        toast.success(t("general.success"), {
+          description: t("products.variant.inventory.toast.itemsManageSuccess"),
+          dismissLabel: t("general.close"),
+        })
+        handleSuccess()
+      },
+      onError: (err) => {
+        toast.error(t("general.error"), {
+          description: err.message,
+          dismissLabel: t("general.close"),
+        })
+      },
+    })
+  })
+
+  return (
+    <RouteFocusModal.Form form={form}>
+      <form
+        className="flex h-full flex-col overflow-hidden"
+        onSubmit={handleSubmit}
+      >
+        <RouteFocusModal.Header>
+          <div className="flex items-center justify-end gap-x-2">
+            <RouteFocusModal.Close asChild>
+              <Button size="small" variant="secondary">
+                {t("actions.cancel")}
+              </Button>
+            </RouteFocusModal.Close>
+            <Button size="small" type="submit" isLoading={isPending}>
+              {t("actions.save")}
+            </Button>
+          </div>
+        </RouteFocusModal.Header>
+        <RouteFocusModal.Body className="flex justify-center">
+          <div className="flex w-full flex-col gap-y-8 px-6 pt-12 md:w-[720px] md:pt-24">
+            <Heading>
+              {t(
+                hasKit
+                  ? "products.create.inventory.heading"
+                  : "fields.inventoryItems"
+              )}
+            </Heading>
+
+            <div className="grid gap-y-4">
+              <div className="flex items-start justify-between gap-x-4">
+                <div className="flex flex-col">
+                  <Form.Label>{variant.title}</Form.Label>
+                  <Form.Hint>{t("products.create.inventory.label")}</Form.Hint>
```

Comment:
rm: It currently says "Inventory kit"
```suggestion
                  <Form.Hint>{t("products.create.inventory.label")}</Form.Hint>
```